src/javascript/binary/reality-check.js

```diff
@@ -165,8 +164,8 @@ RealityCheck = (function ($) {
             $('#reality-check').remove();
         });
 
-        $('#reality-check .blogout').on('click', function () {
-            window.location.href = logout_url;
+        $('#reality-check #btn_logout').unbind('click').click(function(){
+            BinarySocket.send({"logout": "1"});
```

Comment:
Yes, sure. Redirecting to homepage will be handled by `do_logout` function in `page.js` which will be called on the ws response. Our main logout button in header is already doing the same.